ngraph/frontend/onnx_common/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME "onnx_common")
+
+file(GLOB_RECURSE LIBRARY_SRC ${CMAKE_CURRENT_SOURCE_DIR}/src/*.cpp)
+file(GLOB_RECURSE PUBLIC_HEADERS ${CMAKE_CURRENT_SOURCE_DIR}/include/*.hpp)
+
+# Create named folders for the sources within the .vcproj
+# Empty name lists them directly under the .vcproj
+
+source_group("src" FILES ${LIBRARY_SRC})
+source_group("include" FILES ${PUBLIC_HEADERS} ${PUBLIC_HEADERS})
+
+# Create static library
+
+add_library(${TARGET_NAME} STATIC ${LIBRARY_SRC} ${PUBLIC_HEADERS})
+
+if(COMMAND ie_add_vs_version_file)
+        ie_add_vs_version_file(NAME ${TARGET_NAME}
+                        FILEDESCRIPTION "nGraph ONNX Importer common part")
+endif()
```

Comment:
this is not needed for static libraries..

Reply:
right, it is artifact from tests where `onnx_common` was shared